dao/mongodb.rya/src/main/java/org/apache/rya/mongodb/iter/RyaStatementBindingSetCursorIterator.java

```diff
@@ -44,20 +48,21 @@
 
 public class RyaStatementBindingSetCursorIterator implements CloseableIteration<Entry<RyaStatement, BindingSet>, RyaDAOException> {
     private static final Logger log = Logger.getLogger(RyaStatementBindingSetCursorIterator.class);
+    
+    private static final int QUERY_BATCH_SIZE = 50;
```

Comment:
Empirical testing... To support the "show me the first 10" or "first 100" type queries.  I noticed that if we left this at 1000, then the "show me the first 10" or "first 100" type queries were very slow.  However, the "show me all" (where all = 30k), we equally as quick using 50 or 1000.